micropython/examples/badger2040/elimem.py

```diff
@@ -0,0 +1,438 @@
+        d.icon(squares, 1, 144, 24, columns[1], rows[1])
+        d.partial_update(columns[1], 72, 24, 24)
+    elif sq == 9:  # up2
+        activesquares[4][1] = 0
+        d.icon(squares, 1, 144, 24, columns[2], rows[0])
+        d.partial_update(columns[2], 16, 24, 32)
+    elif sq == 7:  # ctri2
+        activesquares[2][1] = 0
+        if activesquares[3][1] == 1:
+            d.icon(squares, 4, 144, 24, columns[2], rows[1])
+        else:
+            d.icon(squares, 5, 144, 24, columns[2], rows[1])
+        d.partial_update(columns[2], 72, 24, 24)
+    elif sq == 8:  # downtri2
+        activesquares[3][1] = 0
+        if activesquares[2][1] == 1:
+            d.icon(squares, 3, 144, 24, columns[2], rows[1])
+        else:
+            d.icon(squares, 5, 144, 24, columns[2], rows[1])
+        d.partial_update(columns[2], 72, 24, 24)
+    d.led(0)
+    pass
+
+
+def activate_square(sq):  # used ONLY to show a new square during gameplay
+    # sq is the square to update
+    d.update_speed(badger2040.UPDATE_SUPER_EXTRA_TURBO)
+    global activesquares
+    if sq == 0:  # a1
+        activesquares[0][0] = 1
+        d.icon(squares, 0, 144, 24, columns[0], rows[2])
+        d.partial_update(columns[0], 96, 24, 32)
+    elif sq == 1:  # b1
+        activesquares[1][0] = 1
+        d.icon(squares, 0, 144, 24, columns[1], rows[2])
+        d.partial_update(columns[1], 96, 24, 32)
+    elif sq == 2:  # c1
+        activesquares[2][0] = 1
+        d.icon(squares, 0, 144, 24, columns[2], rows[2])
+        d.partial_update(columns[2], 96, 24, 32)
+    elif sq == 3:  # down1
+        activesquares[3][0] = 1
+        d.icon(squares, 0, 144, 24, columns[3], rows[1])
+        d.partial_update(columns[3], 72, 24, 32)
+    elif sq == 4:  # up1
+        activesquares[4][0] = 1
+        d.icon(squares, 0, 144, 24, columns[3], rows[0])
+        d.partial_update(columns[3], 16, 24, 32)
+    elif sq == 5:  # a2
+        activesquares[0][1] = 1
+        d.icon(squares, 0, 144, 24, columns[0], rows[1])
+        d.partial_update(columns[0], 72, 24, 24)
+    elif sq == 6:  # b2
+        activesquares[1][1] = 1
+        d.icon(squares, 0, 144, 24, columns[1], rows[1])
+        d.partial_update(columns[1], 72, 24, 24)
+    elif sq == 9:  # up2
+        activesquares[4][1] = 1
+        d.icon(squares, 0, 144, 24, columns[2], rows[0])
+        d.partial_update(columns[2], 16, 24, 32)
+    elif sq == 7:  # ctri2
+        activesquares[2][1] = 1
+        if activesquares[3][1] == 1:
+            d.icon(squares, 2, 144, 24, columns[2], rows[1])
+        else:
+            d.icon(squares, 3, 144, 24, columns[2], rows[1])
+        d.partial_update(columns[2], 72, 24, 24)
+    elif sq == 8:  # downtri2
+        activesquares[3][1] = 1
+        if activesquares[2][1] == 1:
+            d.icon(squares, 2, 144, 24, columns[2], rows[1])
+        else:
+            d.icon(squares, 4, 144, 24, columns[2], rows[1])
+        d.partial_update(columns[2], 72, 24, 24)
+    pass
```

Comment:
You seem to have lots of `pass` calls in various places, which seem to be unnecessary?

Reply:
Yes! I am new to Python and was using those to visualize where my functions/sections ended. As it results in no operation, I figured that was OK- though the interpreter does not ignore them like a comment. At this point I can remove them!